api/unpack/index.go

```diff
@@ -177,8 +182,8 @@ func getEnv(name, def string) string {
 // TODO: consider keeping a LRU cache
 func getNarInfo(key string) (*libstore.NarInfo, error) {
 	path := fmt.Sprintf("%s.narinfo", key)
-	fmt.Println("Fetching the narinfo:", path, "from:", nixCache.CacheURI)
-	r, err := nixCache.GetFile(path)
+	fmt.Println("Fetching the narinfo:", path, "from:", nixCache.URL)
```

Comment:
```suggestion
	fmt.Println("Fetching the narinfo:", path, "from:", nixCache.URL())
```
URL is now a function